swagger/v1/hearing.json

```diff
@@ -278,12 +286,12 @@
         },
         "listingSequence": {
           "description": "Sequence number for the hearing",
-          "example": 1,
+          "example": "1",
```

Comment:
If the type is number, should these be left as Ints?

Reply:
Good spot - have corrected